middleware/authorization.js

```diff
@@ -0,0 +1,101 @@
+const {
+    ACCESS_PRIVATE_KEY,
+    REFRESH_PRIVATE_KEY,
+    ALGORITHM
+} = process.env;
+const jwt = require("jsonwebtoken");
+const userRepository = require("../repository/userRepository");
+const bcrypt = require("bcrypt");
+
+async function issueToken (req, res) {
+    const { body } = req;
+    const row = await userRepository.findUser_email(body.email);
+
+    if (!row) {
+        res.status(400).send( {message: "해당 이메일이 없습니다."});
```

Comment:
```suggestion
        return res.status(400).json( {message: "해당 이메일이 없습니다."});
```

1. 함수 종료를 통해 추가 로직이 수행되지않게 끊어주세요.
2. json메서드를 통해 응답 헤더의 content-type이 application/json이 설정되게 하는걸 추천드립니다.